NASA-AMMOS/aerie-ui
# Update View menu design

resolves #350 
resolves #357

This updates the View dropdown menu to match the new design a little more.

To test:
1. Create a plan and navigate into it
2. Confirm that the initial "Default View" is displayed in the nav bar
3. Open the View menu and click "Save as"
4. Name the new view and click save
5. Confirm that the name of the view is now displayed
6. Change things with the view
7. Confirm that the View menu icon turns yellow
8. Open the View menu and click "Save"
9. Confirm that it turns back to white
10. Confirm that refreshing correctly loads the new state of the view
11. Create a couple of more views
12. Open the View menu and click "Browse saved views"
13. Confirm that a View table opens in a modal
14. Confirm that double clicking or clicking on the "Open View" icon loads the selected view
